broadcast/internal/acceptance_tests/arc_fee_quote_test.go

```diff
@@ -2,147 +2,103 @@ package acceptancetests
 
 import (
 	"context"
-	"errors"
-	"github.com/rs/zerolog"
-	"io"
 	"net/http"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/mock"
+
+	"github.com/jarcoal/httpmock"
```

Comment:
Because of I focused on `http_client` (RequestModel) function with `DoReqest` I tried to test it properly. 
I realized that, these functions/utilities are not properly tested in isolation. Furthermore, the other tests like `acceptance_tests` uses `testify/mock` which do not consider `DoRequest` part and the nature of actual http requests.

That's why I decided to introduce `httpmock` which I find very usefull. Thank's to this lib and changes I made, I found the quite serious bug I described in this PR. 

There is another issue when using `testify/mock` for http-request-related functions. 
It strongly depends on a developer who mocks the `DoRequest` functions whether a test passes or not. And it's more likely that, although tests passes, the actual request during runtime fails. I tried to avoid such scenarios by adding the `httpmock` lib.